### ISSUE TITLE
undefined symbol error during inference  

### ISSUE THREAD

**Issue by AmineDjeghri** [turboderp/exllamav2#98]
i'm trying to run a GPTQ model 'TheBloke/Mistral-7B-Instruct-v0.1-GPTQ/' but i'm getting this error: 

```
ImportError: /home/ubuntu/miniconda3/envs/qback/lib/python3.9/site-packages/exllamav2_ext.cpython-39-x86_64-linux-gnu.so: undefined symbol: _ZN3c104cuda9SetDeviceEi
```
I'm using WSL and installed this wheel : exllamav2-0.0.5+cu118-cp39-cp39-linux_x86_64.whl

**Comment by turboderp** [comment]
I'm not sure, but I'd assume either you don't have CUDA installed, or the version of PyTorch you have doesn't match the CUDA version in your env. Try these two commands:

```
echo "import torch; print(torch.version.cuda)" | python
nvcc --version
```

**Comment by AmineDjeghri** [comment]
![image](https://github.com/turboderp/exllamav2/assets/32715913/b49b4568-033f-45a0-a0b0-2d3544cea7c6)

**Comment by Ph0rk0z** [comment]
For me this was flash attention being compiled with cuda 6.1 GPU visible. Pre ampere is unsupported.. like at all.

**Comment by Ph0rk0z** [comment]
FA2 also doesn't work on windows unless you update cuda to 12.1. Not sure what happens with WSL but it's not exactly linux.

**Comment by turandot2017** [comment]
loader error:
```
Traceback (most recent call last):
  File "/workspace/text-generation-webui/server.py", line 223, in <module>
    shared.model, shared.tokenizer = load_model(model_name)
  File "/workspace/text-generation-webui/modules/models.py", line 79, in load_model
    output = load_func_map[loader](model_name)
  File "/workspace/text-generation-webui/modules/models.py", line 339, in ExLlamav2_loader
    from modules.exllamav2 import Exllamav2Model
  File "/workspace/text-generation-webui/modules/exllamav2.py", line 5, in <module>
    from exllamav2 import (
  File "/usr/local/lib/python3.10/dist-packages/exllamav2/__init__.py", line 3, in <module>
    from exllamav2.model import ExLlamaV2
  File "/usr/local/lib/python3.10/dist-packages/exllamav2/model.py", line 11, in <module>
    from exllamav2.cache import ExLlamaV2Cache
  File "/usr/local/lib/python3.10/dist-packages/exllamav2/cache.py", line 2, in <module>
    from exllamav2.ext import exllamav2_ext as ext_c
  File "/usr/local/lib/python3.10/dist-packages/exllamav2/ext.py", line 14, in <module>
    import exllamav2_ext
ImportError: /usr/local/lib/python3.10/dist-packages/exllamav2_ext.cpython-310-x86_64-linux-gnu.so: undefined symbol: _ZN3c104cuda9SetDeviceEi
root@fe5b514f3bbd:/workspace/text-generation-webui# nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:33:58_PDT_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0

```

**Comment by AmineDjeghri** [comment]
So basically, i was using cuda 11.8 & python 3.9 and i just solved it using cuda 11.2 & python 3.11
i think it might be the cuda version
